docs/DeveloperGuide.md

```diff
@@ -301,169 +364,208 @@ Priorities: High (Must-Have) - `* * *`, Medium (Nice-To-Have) - `* *`, Low (Not
 
 (For all use cases below, the **System** is `CareerSync` and the **Actor** is the `user`, unless specified otherwise)
 
-**Use case: Enter Internship Information From The Main Page**
+**Use Case: Enter Internship Entry From Main Window**
 
 **MSS**
-
-1. User accesses the main page. 
-2. User selects the option to enter internship information. 
-3. System prompts the user to input internship details such as company name, role title, description, etc. 
-4. User inputs the required internship details. 
-5. System validates the input data. 
-6. System saves the internship information. 
+1. User accesses the main page.
+2. User selects the option to add a new internship.
```

Comment:
select option is not an accurate phrasing
maybe "User inputs the add command with internship details such as company name, contact name, etc."

Reply:
remove steps 2 and 3